packages/serve-cli/src/run.ts

```diff
@@ -103,10 +102,9 @@ export async function run({
   productName = 'Mesh',
   productDescription = 'serve GraphQL federated architecture for any API service(s)',
   binName = 'mesh-serve',
-  version,
+  version = 'MESH_VERSION',
```

Comment:
I'd recommend using a defined global instead. This makes it visually more understanding that it's some kind of a global variable that may be replaced. Also works good during developement where it'll be undefined instead of a string.

```suggestion
  version = __VERSION__,
```